src/algorithms/calorimetry/HEXPLIT.h

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023 Sebouh Paul
+
+// An algorithm for splitting calorimeter hits in overlapping cells into "subhits" based on the relative
```

Comment:
Since this is a new algorithm, it would be nice if you could contribute a test for it, see
https://github.com/eic/EICrecon/blob/main/src/tests/algorithms_test/calorimetry_CalorimeterIslandCluster.cc
for an example.